shared/js/initApp.js

```diff
@@ -117,6 +117,16 @@ dashboard.getEditorSource = function() {
 };
 
 /**
+ * Wrapper to allow saveProject to be called by jQuery's trigger()
+ * with a callback param.
+ * @param event {*} Ignored.
+ * @param callback {Function} Callback to pass to saveProject.
+ */
+dashboard.saveProjectWrapper = function(event, callback) {
```

Comment:
Since this is only used once, prefer using an anon function to fix up the args:
```javascript
$(window).on('save_project', function(e, callback) {
  dashboard.saveProject(callback);
});
```